cmake/FairCMakeModulesConfig.cmake.in

```diff
@@ -0,0 +1,15 @@
+################################################################################
+#    Copyright (C) 2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+@PACKAGE_INIT@
+
+set_and_check(@PROJECT_NAME@_PREFIX @PACKAGE_CMAKE_INSTALL_PREFIX@)
```

Comment:
Can you check, if `CMAKE_INSTALL_PREFIX` is always set, even if one does not explicitely set it anywhere?

Reply:
The generated `FairCMakeModulesConfig.cmake` has this:
```cmake
get_filename_component(PACKAGE_PREFIX_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
…
set_and_check(FairCMakeModules_PREFIX ${PACKAGE_PREFIX_DIR})
```
even if one just does a `mkdir build && cd build && cmake ..`.